p2p/simulations/adapters/exec.go

```diff
@@ -407,6 +411,14 @@ func startExecNodeStack() (*node.Node, error) {
 	if err := json.Unmarshal([]byte(confEnv), &conf); err != nil {
 		return nil, fmt.Errorf("error decoding %s: %v", envNodeConfig, err)
 	}
+	// TODO verify that ListenAddr will contain the corrent tcp addr
```

Comment:
Do you plan to eliminate this TODO before merging?

Reply:
I decided to procrastinate it. ExecAdapter is not used anywhere. I'm not sure how easy it is to verify. Do you think it's important to do now?

Reply:
I'm not familiar with the code enough to make that call, rather I would provide the following heuristic to make that decision:
- Would you feel comfortable if we would just remove that `TODO`  from there?
If yes, it's not important to figure that out now. If no, let's figure.

As I've seen `TODO`'s live too long and even causing some (potential) problems - last example in my head is Swarm's [fetcher cancellations](https://github.com/ethersphere/go-ethereum/blob/1a3e25e4c11d09aabd77402585412f1577251f49/swarm/network/fetcher.go#L229) - , I'm just hesitant to leave `TODO`'s to make `master`. Some people how I consider much smarter than myself even bans `TODO`s from `master`.

Reply:
Well we can always make it `context.Background()`

Reply:
corrent -> correct?

Reply:
What do you mean more or less? It's either in use or not. If not maybe we could just remove it instead of spending time on maintenance?  (I see that might cause some mumbling in the team as my changes have been rejected before when I tried to remove unused code. But to me it seems nonsense to spend time on updating unused code.)

Reply:
@frncmx The way I understand it is that the ExecAdapter and the DockerAdapter are related, and the latter we _will_ be using. The fact that it's not in use currently has more to do that we haven't gotten far enough yet with the stability in order to make use of this next step in simulations evolution. I don't think we should remove it.

Reply:
And for the record; the time that was spent on the `ExecAdapter` part of this was minimal. Not much maintenance to speak of, in other words.

Reply:
And with "more or less" I mean that there _is_ a discovery test using `ExecAdapter` which passes. There is, however, no unit test for `ExecAdapter`. And it is currently to my knowledge not in use anywhere else.

Reply:
Okay. I get that we will use later.

Time spent on maintenance: I think time adds up. Also not if someone not familiar with the adapter even a small change can take longer as he or she has to understand the code.

Looking at it from an other angle, now you spend your *little* time on something what *might* be used. While if we remove this functionality and take it back from version control when we really need then we could spend (maybe a little bit more) time on it when we are cetarain we need it.

Reply:
Anyhow the above was just rather on off-topic comment.